src/main/java/io/debezium/connector/yugabytedb/YugabyteDBStreamingChangeEventSource.java

```diff
@@ -659,25 +660,50 @@ protected void maybeWarnAboutGrowingWalBacklog(boolean dispatched) {
 
     @Override
     public void commitOffset(Map<String, ?> offset) {
-        // try {
-        LOGGER.debug("Commit offset: " + offset);
-        ReplicationStream replicationStream = null; // this.replicationStream.get();
-        final OpId commitLsn = null; // OpId.valueOf((String) offset.get(PostgresOffsetContext.LAST_COMMIT_LSN_KEY));
-        final OpId changeLsn = OpId.valueOf((String) offset.get(YugabyteDBOffsetContext.LAST_COMPLETELY_PROCESSED_LSN_KEY));
-        final OpId lsn = (commitLsn != null) ? commitLsn : changeLsn;
-
-        if (replicationStream != null && lsn != null) {
-            if (LOGGER.isDebugEnabled()) {
-                LOGGER.debug("Flushing LSN to server: {}", lsn);
+        try {
+            LOGGER.info("Committing offsets on server");
```

Comment:
Will `StreamingChangeEventSource` also use explicit checkpoints? I assumed implicit checkpoints are sufficient in the default mode.

Reply:
It can, that was already on our roadmap of implementing things for the connector. One thing we can do it add a check/flag in the connector to only call this method when the stream is created in the `EXPLICIT` mode on the server side, otherwise we will not be committing any offsets.